tests/ops/qubit/test_special_unitary.py

```diff
@@ -296,7 +729,7 @@ def test_jax_jit(self):
         jax.config.update("jax_enable_x64", True)
         jnp = jax.numpy
 
-        dev = qml.device("default.qubit", wires=1, shots=None)
+        dev = qml.device("default.qubit", wires=1)
```

Comment:
Why?

Reply:
I noticed it's unnecessary. Can leave it in if preferred :)

Reply:
You're right, didn't realise we were setting shots to `None` 😅